dasharo-compatibility/dcu.robot

```diff
@@ -152,10 +152,7 @@ Prepare DCU Test Environment
     Run    rm -rf dcu
     Run    git clone https://github.com/Dasharo/dcu
 
-    Download To Host Cache
-    ...    logo.bmp
-    ...    https://cloud.3mdeb.com/index.php/s/rsjCdz4wSNesLio/download
-    ...    6e5a6722955e4f78d947654630f27ff833703fbc04776ffed963c96617f6bb2a
+    Copy File    ../osfv-test-data/coreboot_logo_convert.bmp    ${CURDIR}/../dl-cache/logo.bmp
```

Comment:
What is the point of this operation to copy from one local directory to another?